src/screens/ProfileScreen/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { View, ActivityIndicator, Text } from 'react-native';
+import { LOADING, SUCCESS } from '@rootstrap/redux-tools';
+
+import ProfileBackground from 'components/ProfileBackground';
+import ProfileForm from 'components/ProfileForm';
+import useProfile from 'hooks/useProfile';
+import strings from 'locale';
+import { BLACK } from 'constants/colors';
+import common from 'constants/commonStyles';
+
+const ProfileScreen = () => {
+  const { profile, error, status } = useProfile();
+
+  return (
+    <View style={common.formContainer}>
+      <ProfileBackground />
+      {status === LOADING && <ActivityIndicator size="large" color={BLACK} />}
```

Comment:
I think you are going to use loaders in more than one screen so, a good idea it's create a `Loader` component to reuse and you can pass `size` and `color` as props and use these ones as the defaults. WDYT?